portfolio.html

```diff
@@ -2,12 +2,98 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
```

Comment:
Indicate `<meta name="viewport" content="width=device-width, initial-scale=1.0">` in the head of your html, so width can be adjusted by device-width and therefore it will be responsive for various screen-sizes.